cmake/modules/AddCephTest.cmake

```diff
@@ -99,4 +123,18 @@ function(add_tox_test name)
     PATH=${CMAKE_RUNTIME_OUTPUT_DIRECTORY}:${CMAKE_SOURCE_DIR}/src:$ENV{PATH}
     PYTHONPATH=${CMAKE_SOURCE_DIR}/src/pybind)
   list(APPEND tox_test run-tox-${name})
+  set_tests_properties(${test_name} PROPERTIES LABELS "tox")
+  if(DEFINED TOXTEST_COST)
+    set_property(TEST ${test_name} PROPERTY COST ${TOXTEST_COST})
+  else()
+    set_property(TEST ${test_name} PROPERTY COST 25.0)
```

Comment:
Why `25.0`? There are no docs about default `COST` value, but only an [StackOverflow](https://stackoverflow.com/a/48548234) response suggesting is `0`. Why not leaving it undefined?